modules/nextflow/src/main/groovy/nextflow/exception/RetriableException.groovy

```diff
@@ -0,0 +1,7 @@
+package nextflow.exception
+
+/**
+ * Exceptions implementing this interface will always lead to a retry
+ */
+interface RetriableException {
+}
```

Comment:
Sorry I was rash with the naming, we should really call it `TransientException`. Many errors are "retriable", but these "transient" errors are caused by external factors (e.g. spot reclamation) and therefore shouldn't count towards the max retries limit.

Reply:
Okay, done.